src/cfnlint/maintenance.py

```diff
@@ -155,7 +154,7 @@ def patch_spec(content, region):
                     jsonpatch.JsonPatch([all_patch]).apply(content, in_place=True)
                 except jsonpatch.JsonPatchConflict:
                     LOGGER.debug('Patch (%s) not applied in region %s', all_patch, region)
-                except jsonpointer.JsonPointerException:
+                except jsonpatch.JsonPointerException:
```

Comment:
huh, weird that the exception lists switched to `jsonpointer`.  Am I missing something here?  How did you catch this issue?

Reply:
I just went through the list of dependencies in Scallion and noticed this package was there but in code it only appeared here, and it's not listed in setup.py

Reply:
Wish I would have written a test for this exception

Reply:
It is covered by tests though. I am not sure why they passed 🤔

Reply:
I guess that is a good sign.  I was having troubles finding this definition in jsonpatch but it must be as it works.